resources/language/messages.pot

```diff
@@ -2076,3 +2076,31 @@ msgstr ""
 msgctxt "#30657"
 msgid "Disable LSD"
 msgstr ""
+
+msgctxt "#30658"
+msgid "Add extra trackers"
+msgstr ""
+
+msgctxt "#30659"
+msgid "All trackers"
+msgstr ""
+
+msgctxt "#30660"
+msgid "Best trackers"
+msgstr ""
+
+msgctxt "#30661"
+msgid "Minimum set"
+msgstr ""
+
+msgctxt "#30662"
+msgid "Remove original trackers from torrent"
+msgstr ""
+
+msgctxt "#30663"
+msgid "Only for new"
+msgstr ""
+
+msgctxt "#30664"
+msgid "For all"
```

Comment:
I would add "For all torrents", otherwise it can be understood as "For all trackers".

Reply:
this actually is for deprecated approach. new approaches always use either "for all torrents" or "for new torrents" strategy for both add/remove actions. otherwise code will be too tangled (i mean if try to support both options).
i converted this PR to draft while waiting for what approach we will use, so i was planning to revert this commit after that.
but i guess i can do this now as well.